cli/polygonetl/domain/block.py

```diff
@@ -43,3 +43,6 @@ def __init__(self):
 
         self.transactions = []
         self.transaction_count = 0
+
+    def __repr__(self):
```

Comment:
General comment, why the `__repr__`? Not seen this in our other repos. I mean, it makes sense and looks good but just wondering.

Reply:
Just think it's convenient when debugging.
I'm adding it to evmchainetl repo as well in this [PR](https://github.com/nansen-ai/evmchain-etl/pull/143/files)